File: exercicios/Python/1164/1164.py
n = int(input())
p = 'eh perfeito'
pn = 'nao eh perfeito'

for i in range(1,n + 1):
    num = int(input())
    s = 0
    j = 1
    
    while j < num:
        if num % j == 0:
            s = s + j

        j = j + 1
    if s == num:
        print(num,p)
    else:
        print(num,pn)

File: exercicios/JavaScript/1164/1164.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var n = parseInt(lines.shift());

for(let i=0; i<n; i++){
    var num = parseInt(lines.shift()),s=0, j=0;

    while(j<num){
        if(num%j==0){
            s = s + j;
        }
        j += 1;
    }
    if(s==num){
        console.log(num + " eh perfeito");
    }else{
        console.log(num + " nao eh perfeito");
    }
}